File: Palindrome Number/Palindrome Number.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cout<<"Please Enter Number ==> ";
    cin>>n;
    int temp = n;
    int rev = 0;

    while(n>0){
        int rem = n%10;
        rev = (rev * 10) + rem;
        n /= 10;
    }

    if(temp == rev){
        cout<<"Given Number is Palindrome";
    }else{
        cout<<"Given Number is not Palindrome";
    }

    return 0;
}

File: Palindrome Number/Palindrome Number.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 18:34:31 2020

@author: ravi
"""

n = input("Please enter number ==> ")

if int(n) == int(n[::-1]):
    print("Palindrome Number")
else:
    print("Not a Palindrome Number")